WordPress/src/test/java/org/wordpress/android/viewmodel/whatsnew/FeatureAnnouncementViewModelTest.kt

```diff
@@ -29,23 +29,23 @@ class FeatureAnnouncementViewModelTest : BaseUnitTest() {
             FeatureAnnouncementItem(
                     "Test Feature 1",
                     "Test Description 1",
-                    1
+                    "http://wordpress.org"
```

Comment:
minor np: I know it does not make any functional difference and in AS the hint `iconUrl` clarifies, but while looking into the pure code here in GH I was thinking could help to place something like `http://wordpress.org/icon1.png`...but feel free to ignore this 😊.